src/components/Wizard/Inputs/Integer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from "react";
```

Comment:
`import React, { useState } from "react";`

You did this in another file. Do you want to here also?